app/dashboard/models.py

```diff
@@ -310,6 +310,13 @@ class Bounty(SuperModel):
     admin_mark_as_remarket_ready = models.BooleanField(
         default=False, help_text=_('Admin override to mark as remarketing ready')
     )
+    admin_override_org_name = models.CharField(max_length=255, blank=True) # TODO: Remove POST ORGS
```

Comment:
What is this TODO referring to?

Reply:
So ideally if we built out ORGS the right way -> we shall remove this at that point 🙌 
we would do it right way :P 

I leave TODOS in code so that I can fix it up later when the time is right :P

Reply:
👍